nidirect_common/nidirect_common.module

```diff
@@ -237,3 +237,18 @@ function nidirect_common_taxonomy_term_presave(EntityInterface $entity) {
     }
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function nidirect_common_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Add description to theme/subtheme field for landing pages.
+  if (($form_id == 'node_landing_page_form') || ($form_id == 'node_landing_page_edit_form')) {
+    if (isset($form['field_subtheme'])) {
+      $msg = "Note that the theme/subtheme that you choose here will be overridden by this landing page. <br/>";
```

Comment:
Great to see this here. An non-blocking PR enhancement would be to wrap this message in `t()` so it becomes translatable.

Reply:
It is already done in line 251